### ISSUE TITLE
Small Attributions for simple DNN

### ISSUE THREAD

**Issue by theoOmathimatikos** [chr5tphr/zennit#213]
Hi there @chr5tphr,

I was trying to run some simple experiments, using the `epsilon`-rule on a simple DNN (with 4 layers and ReLUs). Yet I encountered an unexpected behavior on the calculated attributions: the values were very small and did not sum up to 1 (for the input, nor intermediate layers).

My code is the following: 

```
from zennit.composites import EpsilonPlus
from zennit.core import Stabilizer
from zennit.attribution import Gradient

# Assume I have a model to load and a data generator
model = simpleDNN(load=True)
data_gen = dataGenerator(dataset='MNIST')

data, label = next(data_gen)


composite = EpsilonPlus()

# First try
with composite.context(model) as modified_model:

    output = modified_model(input)
    attribution, = torch.autograd.grad(
        output, input, grad_outputs=torch.ones_like(output)
    )

# Second try
target = torch.eye(model(data).shape[1])[[label]]
  with Gradient(model=model, composite=composite) as attributor:
      out, relevance = attributor(input, target)
```
Both scores contain values usually of scale 1e-10, and sum up to a value of scale 1e-8. 

I tried all different composites that apply the epsilon rule for linear layers and the scale does not change. Different values for the `epsilon` parameter of the `Stabilizer` result in zero values. 

Am I missing something? What could be wrong?

**Comment by ja-nina** [comment]
Hi There! I am encountering quite a similar problem. I found out the problem is quite possibly occurring in the linear layers: 

![Image](https://github.com/user-attachments/assets/6bfd26a2-c3d4-437f-9058-73e15d2fe636) 

![Image](https://github.com/user-attachments/assets/74c98524-a38f-4349-9c2d-8687faf5d667)

Uploading the relevance values above:) Did you find a solution to this?

**Comment by theoOmathimatikos** [comment]
Hi there @ja-nina,

I have indeed tried passing `bias` in `zero_params` as well as using a `Stabilizer`, and yet the values still remain very low. Not sure what could be wrong.

**Comment by chr5tphr** [comment]
Hey @theoOmathimatikos ,

Could you try to directly use the gradient instead of the `Gradient` attributor? (i.e., with `compositor.context(model)`)
The Gradient attributor has some unintuitive behavior when you do net set the output relevance (which you did not do in your example).
Some people reported issues in the past.

Best,

**Comment by theoOmathimatikos** [comment]
I will try to come back with more code and details asap.
